### PR TITLE
Added example to AffineTransform class

### PR DESCRIPTION
**Pull request by mahamtariq58**
## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

File: skimage/transform/_geometric.py [hunk 1]
```diff
@@ -870,6 +870,38 @@ class AffineTransform(ProjectiveTransform):
...
+    ... [300, 150],
+    ... [150, 400]])
+
+    Creating AffineTransform object
+
+    >>> tform = transform.AffineTransform()
+
+    Computes the transformation matrix
+
+    >>> tform.estimate(src,dst)
```
Comment (lagru) [inline skimage/transform/_geometric.py:899]:
```suggestion
    Estimate the transformation matrix

    >>> tform = transform.AffineTransform()
    >>> tform.estimate(src,dst)
```

**Review by lagru — APPROVED** [review]
Looks good. Thank you @mahamtariq58!

File: skimage/transform/_geometric.py [hunk 2]
```diff
@@ -870,6 +870,31 @@ class AffineTransform(ProjectiveTransform):
...
+    ... [250, 100],
+    ... [150, 200]])
+    >>> dst = np.array([[200, 200],
+    ... [300, 150],
+    ... [150, 400]])
+
+    Estimate the transformation matrix
+
+    >>> tform = transform.AffineTransform()
+    >>> tform.estimate(src,dst)
```
Comment (lagru) [inline skimage/transform/_geometric.py:892]:
```suggestion
    >>> tform.estimate(src,dst)
    True
```

**Comment by stefanv** [comment]
Thanks @mahamtariq58!

**Pull request merged**